src/ttsnake.c

```diff
@@ -887,6 +887,7 @@ int main (int argc, char **argv) {
     free(curr_data_dir);
     free(snake.positions);
 
+    /* Be careful with the killall or we might leave the program without enough time to end, resulting in infinite music */
```

Comment:
You should open a issue to use a proper instruction instead the current one.

Reply:
It might not be necessary anymore with the HTTP request being done. It will certainly die now, I think